concrete-core-fixture/src/fixture/mod.rs

```diff
@@ -139,20 +139,21 @@ pub trait Fixture<Precision: IntegerPrecision, Engine: AbstractEngine, RelatedEn
         repetitions: Repetitions,
         sample_size: SampleSize,
     ) -> bool {
-        (0..repetitions.0)
-            .map(|_| {
-                let repetition_prototypes =
-                    Self::generate_random_repetition_prototypes(parameters, maker);
-                Self::test(
-                    maker,
-                    engine,
-                    parameters,
-                    &repetition_prototypes,
-                    sample_size,
-                )
-            })
-            .reduce(BitAnd::bitand)
-            .expect("At least one repetition is needed.")
+        for _ in 0..repetitions.0 {
+            let repetition_prototypes =
+                Self::generate_random_repetition_prototypes(parameters, maker);
+            let output = Self::test(
+                maker,
+                engine,
+                parameters,
+                &repetition_prototypes,
+                sample_size,
+            );
+            if !output {
+                return false;
+            }
+        }
+        true
```

Comment:
Why do you change this?

Reply:
The original version was not short-circuiting on test failure. This one does !